src/controllers/expenses.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const userService = require('../services/users.js');
+const expenseService = require('../services/expenses.js');
+
+const getAll = (req, res) => {
+  const { userId, from, to, category } = req.query;
+
+  const expenses = expenseService.getAll();
+
+  const foundUser = userService.getUserById(userId);
+
+  const categoryExpenses = expenseService.getExpenseByCategory(category);
+
+  const userExpenses = expenseService.getExpenseByUser(userId);
+
+  const expensesBetweenDates = expenseService.getExpensesBetweenDates(from, to);
```

Comment:
don't need to get expenses by userId, userId or from/to if you don’t have this values